src/components/authentication/Login.tsx

```diff
@@ -31,13 +42,13 @@ const Login: FC = () => {
         <legend>Login</legend>
         <div>
           <Input
-            type="email"
-            value={email}
-            placeholder="Email"
+            type="text"
+            value={username}
+            placeholder="Username"
             required
-            name={"Email"}
+            name={"username"}
             onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
-              setEmail(event.target.value)
+              setUsername(event.target.value)
             }
```

Comment:
This may cause a performance issue. The state will update when user press down one key so the component will keep rerender.

Please optimize it with debounce or other method.

Reply:
Thanks, i choose to use debounce! the only problem i encounter is, it cannot be used with controlled inputs ... not sure if that matters i found some people discussing about it on stackoverflow